index.html

```diff
@@ -1,201 +1,265 @@
+            }
+        };
+    </script>
+    <script src="//1.www.s81c.com/common/stats/ida_stats.js" type="text/javascript"></script>
+</head>
+
+<body>
+    <!-- Navbar -->
+    <nav class="navbar navbar-expand-lg navbar-light fixed-top">
+        <a class="navbar-brand" href="#">
+            <img src="images/global/loopback-full-logo-blue.svg" width=150px>
+        </a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent"
+            aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+        </button>
+
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item">
+                    <a class="nav-link" href="getting-started.html">Getting Started</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="resources.html">Resources</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="contribute.html">Contribute</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="/doc/en/lb4">Docs</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="oasgraph.html">GraphQL</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" href="https://ibm.biz/apitest" target="_blank">Test Your APIs</a>
+                </li>
+                <li class="nav-item" style="padding-top: 10px">
+                    <a class="github-button" href="https://github.com/strongloop/loopback-next" data-icon="octicon-star" data-show-count="true"
+                        aria-label="Star strongloop/loopback-next on GitHub">Star</a>
+                </li>
 
-      <div class="c-grid">
-        <div class="_col -col-3-4">
-          <p>Building on LoopBack's success as an open-source Node.js framework,
-          <a href="https://developer.ibm.com/apiconnect">IBM API Connect</a> provides a graphical tool with many of the API composition features of StrongLoop Arc, a built-in API Micro Gateway,
-          and a CLI with API management and gateway features.</p>
+            </ul>
 
-          <p><b>API Connect Essentials</b> is free to use!  For more information, see <a href="https://developer.ibm.com/apiconnect/getting-started/">API Connect - Getting Started</a>.
-          </p>
         </div>
-      </div>
+    </nav>
+
+    <!--Header alert -->
+    <div class="header-alert">
+        <p class="centered-regular">LoopBack 4 GA (General Availability) has been released in October 2018, read more in <a href="http://strongloop.com/strongblog/loopback-4-ga" target="_blank">the announcement post.</a> Click <a href="/lb3">here</a> for LoopBack 3 pages.</p>
```

Comment:
I've edited the alert to show how to access the LB3 pages.  Not sure if it's necessary.